gdeployfeatures/volume/volume.py

```diff
@@ -15,26 +15,45 @@ def volume_create(section_dict):
     Global.ignore_errors = section_dict.get('ignore_volume_errors')
     section_dict['volname'] = helpers.split_volume_and_hostname(
             section_dict['volname'])
+    if Global.trace:
+        Global.logger.info("Creation of volume successful")
+        Global.logger.info("Splitting volume and hostnames")
     if not section_dict.get('brick_dirs'):
        section_dict = get_common_brick_dirs(section_dict)
+       if Global.trace:
+           Global.logger.info("Retrieving common brick directories among hosts.")
     else:
         section_dict = validate_brick_dirs(section_dict, 'brick_dirs')
+        if Global.trace:
+            Global.logger.info("Error in retrieving brick directories"\
+                               "Validating brick directories.")
```

Comment:
A space after before validating is needed. Else the message will look like directoriesvalidating ...